README.md

```diff
@@ -14,20 +14,20 @@ mackerel-client-go is a Go client library for [mackerel.io API](https://mackerel
 # Usage
 
 ```go
-import mkr "github.com/mackerelio/mackerel-client-go"
+import "github.com/mackerelio/mackerel-client-go"
 ```
 
 ```go
-client = mkr.NewClient("<Put your API key>")
+client = mackerel.NewClient("<Put your API key>")
```

Comment:
User may use `:=` for defining the client.

Reply:
Fixed at 
https://github.com/mackerelio/mackerel-client-go/pull/74/commits/5ea755dd5c2d3eb6812289b067c9703fd41afa5b .